src/Lykke.Service.OperationsHistory/Modules/ServiceModule.cs

```diff
@@ -93,6 +105,28 @@ private void RegisterAzureRepositories(ContainerBuilder builder)
         private void RegisterServiceClients(ContainerBuilder builder)
         {
             builder.RegisterLykkeServiceClient(_settings.ClientAccountServiceClient.ServiceUrl);
+            builder.RegisterInstance<IAssetsService>(
```

Comment:
Is it necessary to create an instance here? Why not just register like it done in the line 114